### PR TITLE
fix: Keithley_2400 READ? format may be inconsistent

### PR DESCRIPTION
**Pull request by lychee512**
Fixes 'bugginess' mentioned in #488 

Changes proposed in this pull request:
- Added code to ensure Keithley 2400 instrument sends responses to READ? in correct format in the driver __init__ method.
- Modify 'output' parameter to use create_on_off_val_mapping instead of int.

Breaking changes: 'output' parameter type change could possibly break existing code that depends on 'output' being an int.

@WilliamHPNielsen

### CODE REVIEW

File: qcodes/instrument_drivers/tektronix/Keithley_2400.py [hunk 1]
```diff
@@ -97,6 +98,7 @@ def __init__(self, name, address, **kwargs):
                            docstring="Measure resistance from current and voltage "
                                      "Note that it is an error to read current "
                                      "and voltage with output off")
+        self.write(':TRIG:COUN 1;:FORM:ELEM VOLT,CURR')
```
Comment (jenshnielsen) [inline qcodes/instrument_drivers/tektronix/Keithley_2400.py]: Is this the same mode that reset would put the instrument in? Could you document (for instance in a comment) what this does

Comment (lychee512) [reply qcodes/instrument_drivers/tektronix/Keithley_2400.py]: Comment added in the driver itself, and pushed in a new commit. Did you mean to comment just here instead? I didn't realise there was a github comment feature!

Comment (jenshnielsen) [reply qcodes/instrument_drivers/tektronix/Keithley_2400.py]: Thanks

Comment (astafan8) [reply qcodes/instrument_drivers/tektronix/Keithley_2400.py]: both these commands aught to be represented as qcodes parameters (or at least methods), in which case this explanation wouldn't be needed. but we can leave this to another PR.

**Pull request merged**